packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -61,7 +61,49 @@ No. You can only set up one custom domain per Okta org.
 
 Yes. When you turn the custom domain on, the Okta domain (for example, `example.okta.com`) still works.
 
-## Validate your TLS certificate
+<ApiLifecycle access="ea" />
+
+## Configure a custom domain via Okta-managed certificates
+
+This method of configuring a custom domain is recommended because Okta will manage your cerificate renewals in imperpetuity, via an integration with Let's Encrypt, a free certificate authority. The certificate procurement process is free, and also faster and easier than configuring a custom domain with your own certificate.
+
+1. In the Admin Console, select **Customizations**, and then **Domain**.
+2. In the **Custom URL Domain** box, click **Edit**.
+3. Click **Get Started** to start the configuration wizard.
+
+### Add your subdomain information
+
+On the Add Domain page of the configuration wizard, enter your subdomain name, for example, `login.example.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
+
+### Create a DNS TXT  and CNAME record
+
+You need to add a DNS TXT and CNAME record to your domain to verify ownership of your domain with Okta before Okta can serve traffic over it. This record includes the Okta-generated values provided in the **Host** and **Value** columns of the table on the **Update your DNS page**. Okta verifies that you own your domain when it finds the TXT record that contains the generated value.
+
+1. On the **Update your DNS** page of the configuration wizard, copy the values of the **Host** and **Data** columns into a text file.
+
+2. Sign in to your Domain Name registrar and locate the option to modify your DNS records.
+
+3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field.
+
+> **Note**: Depending on your domain provider, you may only need to enter `_oktaverification` rather than `_oktaverification.login.example.com`. If your domain provider doesn't support the value that you enter, verification fails and your custom URL domain configuration is incomplete.
```

Comment:
I'm not sure how to word this but for BYOC, the TXT record will begin with _oktaverification but with Okta-managed, the record will begin with _acme-challenge (because that record name/value is provided by the ACME server.